certified-connectors/WithoutWire Inventory Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1204 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "WithoutWire Inventory Platform",
+    "description": "The connector contains methods to connect WithoutWire for move requests, order fullfillment requests, adjustments, audits, and allocation as well as any inventory management needs",
+    "version": "1.0",
+	"contact": {
+		"name": "WithoutWire Support",
+		"url":"https://www.withoutwire.com/knowledgebase",
+		"email":"support@withoutwire.com"
+	}
+  },
+  "host": "api.withoutwire.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/integration/workorder": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default"
+		  },
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "WarehouseName": {
+                    "type": "string",
+                    "description": "WarehouseName"
+                  },
+                  "CustomerPoNumber": {
+                    "type": "string",
+                    "description": "CustomerPoNumber"
+                  },
+                  "VendorShipmentNumber": {
+                    "type": "string",
+                    "description": "VendorShipmentNumber"
```

Comment:
@PavloNikulinEnavate Can you please provide valid description values (free from grammatical and spelling errors) for all the properties as they are visible to public.

Documentation - [Define operation descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)